classification1.Rmd

```diff
@@ -729,102 +833,143 @@ yourself. You will see further on in Section
-    xend = unlist(neighbours[3, attrs[1]]),
-    yend = unlist(neighbours[3, attrs[2]])
-  ), color = "black")
+    xend = unlist(neighbors[3, attrs[1]]),
+    yend = unlist(neighbors[3, attrs[2]])
+  ), color = "black", size = 0.5)
 
 # create new scaled obs and get NNs
 new_obs_scaled <- tibble(Area = -0.72, Smoothness = 2.8, Class = "unknown")
 my_distances_scaled <- table_with_distances(scaled_cancer[, attrs], new_obs_scaled[, attrs])
-neighbours_scaled <- scaled_cancer[order(my_distances_scaled$Distance), ]
+neighbors_scaled <- scaled_cancer[order(my_distances_scaled$Distance), ]
 
 # add to the df
 scaled_cancer <- bind_rows(scaled_cancer, new_obs_scaled)
 
 # plot the scatter
-scaled <- ggplot(scaled_cancer, aes(x = Area, y = Smoothness, group = Class, color = Class, shape = Class)) +
+scaled <- ggplot(scaled_cancer, aes(x = Area, y = Smoothness, 
+                                    group = Class, color = Class, shape = Class, size = Class)) +
   geom_point(alpha = 0.6) + 
   scale_color_manual(name = "Diagnosis", 
                      labels = c("Benign", "Malignant", "Unknown"), 
                      values = c("steelblue2", "orange2", "red")) +
   scale_shape_manual(name = "Diagnosis", 
                      labels = c("Benign", "Malignant", "Unknown"),
-                     values= c(16, 16, 17)) +
+                     values= c(16, 16, 18)) +
+  scale_size_manual(name = "Diagnosis", 
+                     labels = c("Benign", "Malignant", "Unknown"),
+                    values=c(2,2,2.5)) + 
   ggtitle("Standardized Data") +
+  labs(x = "Area (standardized)", y = "Smoothness (standardized)") + 
   # coord_equal(ratio = 1) +
   geom_segment(aes(
     x = unlist(new_obs_scaled[1]), y = unlist(new_obs_scaled[2]),
-    xend = unlist(neighbours_scaled[1, attrs[1]]),
-    yend = unlist(neighbours_scaled[1, attrs[2]])
-  ), color = "black") +
+    xend = unlist(neighbors_scaled[1, attrs[1]]),
+    yend = unlist(neighbors_scaled[1, attrs[2]])
+  ), color = "black", size = 0.5) +
   geom_segment(aes(
     x = unlist(new_obs_scaled[1]), y = unlist(new_obs_scaled[2]),
-    xend = unlist(neighbours_scaled[2, attrs[1]]),
-    yend = unlist(neighbours_scaled[2, attrs[2]])
-  ), color = "black") +
+    xend = unlist(neighbors_scaled[2, attrs[1]]),
+    yend = unlist(neighbors_scaled[2, attrs[2]])
+  ), color = "black", size = 0.5) +
   geom_segment(aes(
     x = unlist(new_obs_scaled[1]), y = unlist(new_obs_scaled[2]),
-    xend = unlist(neighbours_scaled[3, attrs[1]]),
-    yend = unlist(neighbours_scaled[3, attrs[2]])
-  ), color = "black")
+    xend = unlist(neighbors_scaled[3, attrs[1]]),
+    yend = unlist(neighbors_scaled[3, attrs[2]])
+  ), color = "black", size = 0.5)
 
 gridExtra::grid.arrange(unscaled, scaled, ncol = 2)
 ```
 </center>
 
+```{r 05-scaling-plt-zoomed, fig.height = 4, fig.width = 10, echo = FALSE, fig.cap = "Close up of three nearest neighbors for unstandardized data"}
```

Comment:
This is a super cool plot. The only comment I have here is that the grey highlight showing where the "zoomed in" region is is a bit unclear.